### DISCUSSION TITLE
Вопрос по коду действия таблицы

### DISCUSSION THREAD

**Discussion by ivev7** [totumonline/totum-ru-issues-and-discussions#555] [Как это сделать?] [answered]
Доброго времени суток!
В коде действия таблицы  я написал такое:
```
=: if(condition: $data_zapreta > #data; then: $block)
data_zapreta: select(table: 'proekty'; field: 'data_zapreta_redaktirovaniya'; where: 'id' = #proekt)
block: errorExeption(text: "Нельзя изменять Контакт в закрытом периоде!") 
```

 смысл такой: по оригинальному значению поля Проект мы выбираем Дату блокировки из другой таблицы. Затем сравниваем ее с оригинальной датой из поля Дата и если она раньше Даты блокировки, то не даем сохранить, но оно не робит

**Comment by ttm-alexeypolunin** [comment]
Смотрите отладку. Тк это код действия таблицы — то удостоверьтесь, что #proekt и #data не относятся к строчной части (по названию относятся)

**Reply by ivev7** [reply]
Они и есть из строковой. Так тут и вопрос , как тогда мне запретить редактирование строк в таблице раннее заданной даты?  Возможно ли такое?  Спасибо за ответ.

**Reply by erpic-ru** [reply, marked as answer]
Можно через форматирование строк в коде форматирования таблицы по условию setrowformat block: true

**Reply by ivev7** [reply]
Работает спасибо
